src/Turbo/src/Twig/TwigExtension.php

```diff
@@ -12,19 +12,24 @@
 namespace Symfony\UX\Turbo\Twig;
 
 use Psr\Container\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Mercure\Authorization;
 use Symfony\UX\Turbo\Bridge\Mercure\TopicSet;
 use Twig\Environment;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFunction;
 
 /**
  * @author Kévin Dunglas <kevin@dunglas.fr>
+ * @author Pierre Ambroise <pierre27.ambroise@gmail.com>
  */
 final class TwigExtension extends AbstractExtension
 {
     public function __construct(
         private ContainerInterface $turboStreamListenRenderers,
         private string $default,
+        private ?Authorization $authorization = null,
+        private ?RequestStack $requestStack = null,
```

Comment:
We should then declare a Twig Runtime .. as this should probably not be in the hot path.